### PR TITLE
[weather.metoffice] 2.0.2

### PR DESCRIPTION
**Pull request by powlo**
### Description
Updates weather.metoffice for leia with minor fixes.
### Checklist:
- [X ] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [ X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [ X] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

### CODE REVIEW

File: weather.metoffice/addon.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  <extension point="xbmc.addon.metadata">
+    <assets>
+      <icon>icon.png</icon>
+      <fanart>fanart.jpg</fanart>
+    </assets>
+    <summary lang="en">UK weather forecast from the Met Office.</summary>
+    <description lang="en">UK weather forecast provided by Met Office (http://www.metoffice.gov.uk/)</description>
+    <platform>all</platform>
+    <language>en</language>
+    <license>GNU GPL v2</license>
```
Comment (enen92) [inline weather.metoffice/addon.xml:18]: GPL-2.0-only or GPL-2.0-or-later

File: weather.metoffice/addon.xml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    <forum>https://forum.kodi.tv/showthread.php?tid=166362</forum>
+    <website>https://www.metoffice.gov.uk/services/data/datapoint</website>
+    <source>https://github.com/powlo/weather.metoffice</source>
+    <email></email>
```
Comment (enen92) [inline weather.metoffice/addon.xml:22]: Drop if empty

File: weather.metoffice/addon.xml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  <extension point="xbmc.python.weather" library="src/metoffice/default.py" />
+  <extension point="xbmc.addon.metadata">
+    <assets>
+      <icon>icon.png</icon>
+      <fanart>fanart.jpg</fanart>
+    </assets>
+    <summary lang="en">UK weather forecast from the Met Office.</summary>
+    <description lang="en">UK weather forecast provided by Met Office (http://www.metoffice.gov.uk/)</description>
+    <platform>all</platform>
+    <language>en</language>
```
Comment (enen92) [inline weather.metoffice/addon.xml:17]: Remove language tag, only applies to media (not weather)

File: weather.metoffice/addon.xml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    <import addon="script.module.pytz" version="2014.2" />
+    <import addon="script.module.pil" />
+  </requires>
+  <extension point="xbmc.python.weather" library="src/metoffice/default.py" />
+  <extension point="xbmc.addon.metadata">
+    <assets>
+      <icon>icon.png</icon>
+      <fanart>fanart.jpg</fanart>
+    </assets>
+    <summary lang="en">UK weather forecast from the Met Office.</summary>
```
Comment (enen92) [inline weather.metoffice/addon.xml:14]: en_GB

File: weather.metoffice/addon.xml [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+    <description lang="en">UK weather forecast provided by Met Office (http://www.metoffice.gov.uk/)</description>
```
Comment (enen92) [inline weather.metoffice/addon.xml:15]: en_GB

File: weather.metoffice/exclude.lst [hunk 6]
```diff
@@ -0,0 +1,2 @@
+*.pyc
```
Comment (enen92) [inline weather.metoffice/exclude.lst:1]: Remove from submissiom

File: weather.metoffice/src/metoffice/utilities.py [hunk 7]
```diff
@@ -0,0 +1,171 @@
...
+                e.args = ('Error',)
+            if len(e.args) == 1:
+                e.args = e.args + ('See log file for details',)
+            if xbmcgui.getCurrentWindowId() == WEATHER_WINDOW_ID or xbmcgui.getCurrentWindowId() == ADDON_BROWSER_WINDOW_ID:
+                args = (e.args[0].title(),) + e.args[1:4]
+                DIALOG.ok(*args)  # @UndefinedVariable
+    return wrapper
+
+
+def xbmcbusy(f):
```
Comment (enen92) [inline weather.metoffice/src/metoffice/utilities.py:40]: This doesn't work anymore, use dialogbusynocancel

**Review by enen92 — CHANGES_REQUESTED** [review]

**Comment by powlo** [comment]
@enen92 thanks for the PR feedback. Do you prefer fixes to be in a rebase (so keeping the "one commit" rule), or a new commit containing the fixes?

**Comment by enen92** [comment]
I prefer actually a new commit to avoid having to go over all the code again. We'll squash when merging, thanks

**Comment by powlo** [comment]
@enen92 see fixes in f3aa61f.

**Review by enen92 — APPROVED** [review]

**Pull request merged**